charts/fluentd/templates/_helpers.tpl

```diff
@@ -47,4 +47,10 @@ Full image name.
 </assume_role_credentials>
 s3_bucket {{ .Values.s3.s3_bucket }}
 s3_region {{ .Values.s3.s3_region }}
+{{- if .Values.s3.path -}}
```

Comment:
@furminsky Do we need the 'if' conditions here or will the values be ignored if they are left empty?